definitions/npm/react-modal_v3.12.x/flow_v0.104.x-/react-modal_v3.12.x.js

```diff
@@ -0,0 +1,67 @@
+declare module 'react-modal' {
+  declare type ModalElementProps = {
+    ref: (node: ?HTMLElement) => mixed,
+    id: string,
+    className: string,
+    style: { [key: string]: string | number },
+    onClick: (event: SyntheticMouseEvent<*>) => void,
```

Comment:
The existential type `*` is deprecated, probably `any` is enough here.

Reply:
Event callback returns types are usually better typed as `mixed`, accept async callbacks.